ISSUE_TEMPLATE.md

```diff
@@ -1,17 +1,61 @@
+# Issue Template
+
+Please use this Template to open Issues in this Repository.
+
+```markdown
 Steps to reproduce
 ==================
+
 1. Do this
 2. Do that
 3. Do here
 
+
 Issues
 ======
 * While step 2: that and that is happening
 
+
 Expected
 ========
 * While step 2: that and that should happen
 
+
+Additional Notes
```

Comment:
Yes i was considering to remove the === as i did't like it myself but did not want to change your template to much :) i will change it